landing/index.html

```diff
@@ -2,12 +2,178 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Document</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mentors International | Home</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <link
+      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700,900|Poppins:400,500,600,700,800,900"
```

Comment:
This is importing a lot of font weights.. each font weight is an additional font that needs to be downloaded, can you reduce this to the minimal font weights needed?